pkgs/by-name/ka/katrain/package.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+let
+  kivymd_0_104_1 = python3.pkgs.kivymd.overridePythonAttrs (old: {
+    src = (
+      old.src.override {
+        rev = "0.104.1";
+        hash = "sha256-qWiZ5bKw4gQtTvhe4w0Cadk17TbFFQWC0hvSFz9K4xA=";
+      }
+    );
+  });
+in
+python3.pkgs.buildPythonApplication rec {
+  pname = "katrain";
+  version = "v1.17.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "sanderland";
+    repo = "katrain";
+    tag = version;
+    hash = "sha256-v0VGjxayXLbBSXs/jkYeXgbyDg+JFLgxESDuh/qbg1U=";
```

I'm getting a different hash here

```suggestion
    hash = "sha256-2DHamhxsOPKldRVxKHUoGKZn1Ao9l4D+TB4NkroixbA=";
```

---

thanks, I had this cached somehow from when i bumped 1.16 -> 1.17